account_invoice_rounding_by_currency/res_company.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_res_company_currencies_settings" model="ir.ui.view">
+            <field name="name">res.company.add.rounding.correncies</field>
+            <field name="model">res.company</field>
+            <field name="priority">50</field>
+            <field name="inherit_id" ref="base.view_company_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//group[@name='account_grp']" position="after">
+                <group name="account_grp_currency" string="Currencies Rounding Rules">
+                    <field name="currency_rounding_rules" nolabel="1">
```

Comment:
See on
http://3117029-108-0247b6.runbot1.odoo-community.org/web?db=3117029-108-0247b6-base
the field goes on the right.

Moreover, is it possible to have this settings along with the main ones? Under settings -> configuration -> invoicing ?